lte/gateway/c/core/oai/tasks/amf/amf_recv.cpp

```diff
@@ -484,8 +504,66 @@ int amf_handle_registration_request(
       }
     } else if (msg->m5gs_mobile_identity.mobile_identity.guti
                    .type_of_identity == M5GSMobileIdentityMsg_GUTI) {
+      for (uint8_t i = 0; i < amf_config.guamfi.nb; i++) {
+        if ((msg->m5gs_mobile_identity.mobile_identity.guti.mcc_digit2 ==
```

Comment:
Can we define a macro for this comparison?

Reply:
Added macro